lib/Cake/Network/CakeResponse.php

```diff
@@ -68,6 +68,7 @@ class CakeResponse {
 		415 => 'Unsupported Media Type',
 		416 => 'Requested range not satisfiable',
 		417 => 'Expectation Failed',
+		429 => 'Too Many Requests',
```

Comment:
@lorenzo He had made the commits on top of 2.6 branch and the PR is opened against 2.7, that might be the reason of the wacky diff.